examples/data-sources/bloxone_dhcp_fixed_addresses/data-source.tf

```diff
@@ -0,0 +1,16 @@
+# Get DHCP fixed address filtered by an attribute
+data "bloxone_dns_views" "dhcp_fixed_address_by_attribute" {
```

Comment:
`dhcp_fixed_address` in `dhcp_fixed_address_by_attribute` is redundant. Also for consistency, will prefer if you name it `example_by_attribute`. Same comment for all the resources in the example.